setup.py

```diff
@@ -0,0 +1,15 @@
+# -*- coding: utf-8 -*-
+from setuptools import setup
+
+
+setup(
+    name="linked_list",
```

Comment:
The name of the package should probably be something like `data-structures` considering that far more than just a linked-list is contained within.